src/lib/mooncat-parser.ts

```diff
@@ -75,6 +75,9 @@ export class MooncatParser {
     // 26 values between: 120 and 145 --> 26/256 --> 0.1015625 --> ~10%
     const orangeBackground = bytes[6] >= 120 && bytes[6] <= 145;
 
+    // 10% chance of crown
+    const crown = bytes[7] >= 120 && bytes[7] <= 145;
```

Comment:
hmm... I was thinking of making this an off-chain trait. the cat gets a crown if its within a numbered range.
eg. first 1000 cats get a gold crown but only if we click in the image. so every cat will have two images:
- normal image: needs only transactionId + blockId
- 2nd image: unlocks if we know the cat's number

this aligns with the current way of how are collections are done. they are all off-chain.

Reply:
@Ethspresso what's you opionion about this idea?

Reply:
Sounds good to me. Will you adjust the code?

Reply:
yep! 🫡